Assets/MixedRealityToolkit/Utilities/WindowsApiChecker.cs

```diff
@@ -33,37 +36,64 @@ public static void CheckApiContracts()
             UniversalApiContractV5_IsAvailable = false;
             UniversalApiContractV4_IsAvailable = false;
             UniversalApiContractV3_IsAvailable = false;
-#endif
+#endif // WINDOWS_UWP
+#pragma warning restore 0618
+        }
+
+        /// <summary>
+        /// Checks to see if the requested method is present on the current platform.
+        /// </summary>
+        /// <param name="namespaceName">The namespace (ex: "Windows.UI.Input.Spatial") containing the class.</param>
+        /// <param name="className">The name of the class containing the method (ex: "SpatialInteractionMananger").</param>
+        /// <param name="methodName">The name of the method (ex: "IsSourceKindSupported").</param>
+        /// <returns>True if the method is available and can be called. Otherwise, false.</returns>
+        public static bool IsMethodAvailable(
+            string namespaceName,
+            string className,
+            string methodName)
+        {
+#if WINDOWS_UWP
```

Comment:
I think we can call this from the DotNetWinRT APIs too, but that can be added later if needed